### ISSUE TITLE
Release Files and Release management

### ISSUE THREAD

**Issue by anivar** [nic-delhi/AarogyaSetu_Android#36]
Please add binary releases corresponding to current production versions. 
It will be good if release management can be done directly from this repo to play store https://github.com/codepath/android_guides/wiki/Automating-Publishing-to-the-Play-Store

**Comment by srikanthlogic** [comment]
+1, also tag the commit used for the release build.

**Comment by FarmaanElahi** [comment]
Yeah It would be good if a CI/CD pipelines is setup. It will definitely save in release time and those want to want see the binaries will have access. It's very common open source pattern

**Comment by anivar** [comment]
Related  solid proposal https://github.com/nic-delhi/AarogyaSetu_Android/issues/186
